informing/internal/models/rate.go

```
@@ -0,0 +1,18 @@
+package models
+
+import (
+	"github.com/google/uuid"
+	"gorm.io/gorm"
+	"time"
+)
+
+type Rate struct {
+	ID        uuid.UUID `gorm:"primaryKey" json:"id"`
+	Rate      float64   `json:"rate"`
+	CreatedAt time.Time `json:"createdAt"`
+}
+
+func (r *Rate) BeforeCreate(tx *gorm.DB) (err error) {
+	r.ID, _ = uuid.NewUUID()
```

You have an error in your function, maybe we should return an error from uuid.NewUUID() ?)

---

yeap, good one, the error initially were added to implement gorm BeforeCreate interface, will fix that